.travis.yml

```
@@ -3,9 +3,10 @@ language: node_js
 notifications:
   email: false
 node_js:
-  - '10'
-  - '12'
-  - '14'
+  - 10
+  - 12
+  - 14
```

Node 15 is still supported until June 1
```suggestion
  - 14
  - 15
```

---

We shouldn't support odd Node versions, only when they're `latest`

---

As far as I understand from [Node's LTS schedule](https://github.com/nodejs/Release), 15 is actually still supported right now, even though 16 (which will eventually be upgraded to an LTS) has been released since. Technically 16 is the only latest release here anyway.